pkgs/tools/filesystems/irods/common.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, bzip2, zlib, autoconf, automake, cmake, gnumake, help2man , texinfo, libtool , cppzmq , libarchive, avro-cpp, boost, jansson, zeromq, openssl, pam, libiodbc, kerberos, gcc, libcxx, cross, which }:
+
+# Common attributes of irods packages
+
+with stdenv;
+
+# Boost and avro-c++ have to be built with the same stdenv
```

Comment:
I might be doing something wrong, but if I remove this part, I get boost and avro c++ from the gcc build in my dependencies, instead of an llvm build.